client/src/components/ui/TextInput.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+  import { InputType } from '../types.ts';
+  export let type: InputType = InputType.Primary;
```

Comment:
TypeScript is smart enough to infer the type annotation.
```suggestion
  export let type = InputType.Primary;
```